provisioning/device/src/constants.ts

```diff
@@ -23,7 +23,9 @@ export class ProvisioningDeviceConstants {
   /**
    * apiVersion to use while communicating with service.
    */
-  static apiVersion: string = '2019-03-31';
+  // static apiVersion: string = '2019-03-31';
+  // To be used for DPS Cert Management
+  static apiVersion: string = '2021-11-01-preview';
```

Comment:
Remove the commented above two lines.

Reply:
added a TODO so that we can change the version after release